config/environment.js

```diff
@@ -32,7 +32,7 @@ module.exports = function (environment) {
         'acmidm-oauth2': {
           apiKey: '{{OAUTH_API_KEY}}',
           baseUrl: '{{OAUTH_BASE_URL}}',
-          scope: 'openid vo profile abb_besluitendatabank',
+          scope: 'openid vo profile abb_toezichtsdatabank',
```

Comment:
Do the scope needs to be changed for the right openid profile ?

Reply:
You can change this abb_eredienstbesluitendatabank

Reply:
will only be functional waaay later

Reply:
Okay thanks :+1: